opm/parser/eclipse/EclipseState/Schedule/Group.hpp

```diff
@@ -114,6 +115,7 @@ namespace Opm {
         std::shared_ptr<GroupProduction::ProductionData> m_production;
         std::shared_ptr<DynamicState<std::shared_ptr< const WellSet >> > m_wells;
         std::shared_ptr<DynamicState<bool> > m_isProductionGroup;
+        std::shared_ptr<DynamicState<bool> > m_isInjectionGroup;
```

Comment:
This is perhaps a stupid question, but does this really have to be a `shared_ptr`?  Or could this (and `m_isProductionGroup`) be a `unique_ptr` or (better yet) a pure value?